wandb/sdk/launch/sweeps/scheduler.py

```diff
@@ -681,8 +684,8 @@ def _add_to_launch_queue(self, run: SweepRun) -> bool:
             entity=self._entity,
             queue_name=self._kwargs.get("queue"),
             project_queue=self._project_queue,
-            resource=self._kwargs.get("resource", None),
-            resource_args=self._kwargs.get("resource_args", None),
+            resource=_job_launch_config.get("resource"),
+            resource_args=_job_launch_config.get("resource_args"),
```

Comment:
should we fall back to the current resource args? and resource

Reply:
when resource or resource_args are None, we get the resource args from the queue. you mean from the scheduler.resource_args? See discussion with Nick above

Reply:
This `self._kwargs.get("resource_args", None)` predates default resource configs I think